### PR TITLE
Completed validation_data entry for .fit()

### PR DESCRIPTION
**Pull request by WurmD**

### CODE REVIEW

**Comment by WurmD** [comment]
@googlebot I signed it!

**Comment by gbaned** [comment]
@WurmD Can you please resolve conflicts? Thanks!

**Comment by tqa236** [comment]
Is this the problem of this PR or the problem of the CIs? It looks like that the CIs are stuck.

**Comment by gbaned** [comment]
> Is this the problem of this PR or the problem of the CIs? It looks like that the CIs are stuck.

@tqa236 It is waiting for review. Thanks!

File: tensorflow/python/keras/engine/training.py [hunk 1]
```diff
@@ -722,6 +722,8 @@ def fit(self,
               - tuple `(x_val, y_val)` of Numpy arrays or tensors
               - tuple `(x_val, y_val, val_sample_weights)` of Numpy arrays
               - dataset
+              - A generator or `keras.utils.Sequence` returning `(inputs, targets)`
+            or `(inputs, targets, sample_weights)`.
             For the first two cases, `batch_size` must be provided.
             For the last case, `validation_steps` could be provided.
             Note that `validation_data` does not support all the data types that
```
Comment (fchollet) [inline tensorflow/python/keras/engine/training.py:729]: Please remove this statement which is no longer accurate.

**Review by fchollet — CHANGES_REQUESTED** [review]
Thank you for the PR!

**Comment by gbaned** [comment]
@WurmD  Can you please check @fchollet's comments and keep us posted ? Thanks!

File: tensorflow/python/keras/engine/training.py (hunk 1 above)
Comment (WurmD) [reply tensorflow/python/keras/engine/training.py:729]: which statement @fchollet ?
And the whole pull request is a single line, if  that line is now inadequate close the pull request?\

(plus, this simple PR was made 6 months ago, and was failed to be merged, that suggests a necessary review of your teams process @gbaned )

**Comment by vr140** [comment]
any update on this? 

according to https://github.com/tensorflow/tensorflow/issues/39797,  validation_data CAN in fact be a Sequence object?   if so, should we merge this docstring update and amend the last sentence in the param documentation:

"Note that `validation_data` does not support all the data types that are supported in `x`, eg, dict, generator or `keras.utils.Sequence`."

**Comment by fchollet** [comment]
We're fixing the docstring as part of a separate change. Sorry for the confusion with the initial review, I meant that the statement *"Note that validation_data does not support all the data types that are supported in x, eg, dict, generator or keras.utils.Sequence"* was no longer accurate.

**Pull request closed without merging**